gulpfile.coffee

```diff
@@ -1,4 +1,5 @@
 gulp = require 'gulp'
+module.exports = gulp
```

Comment:
It apparently lets node know about gulp? Idk, I needed it for sublime-gulp as well as gulpfile.js.